Author: Svante Signell <svante.signell@telia.com>
Description: Define PATH_MAX to fix build on the Hurd.
Forwarded: no

--- gimp-2.6.11/libgimpbase/gimpreloc.c	2010-07-03 00:51:56.000000000 +0200
+++ gimp-2.6.11.modified/libgimpbase/gimpreloc.c	2011-09-09 08:12:09.000000000 +0200
@@ -27,6 +27,9 @@
 
 #include "gimpreloc.h"
 
+#ifndef PATH_MAX
+#define PATH_MAX 4096
+#endif
 
 /*
  * Find the canonical filename of the executable. Returns the filename
